src/main/java/frc/utils/digitalinput/chooser/ChooserDigitalInput.java

```diff
@@ -0,0 +1,45 @@
+package frc.utils.digitalinput.chooser;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.utils.digitalinput.DigitalInputInputsAutoLogged;
+import frc.utils.digitalinput.IDigitalInput;
+
+public class ChooserDigitalInput implements IDigitalInput {
+
+	private final SendableChooser<Boolean> isTrueSendableChooser;
+	private boolean inverted = false;
+
+	public ChooserDigitalInput(boolean inverted) {
```

Comment:
i dont think you should have inverted here, because you pick the value in the dashboard